src/sidebar/components/test/markdown-editor-test.js

```diff
@@ -97,6 +97,11 @@ describe('MarkdownEditor', () => {
     },
   ];
   commands.forEach(({ command, key, effect }) => {
+    describe('renders appropriate tooltip for user OS', () => {
+      it('should show "Cmd-shortcutKey" for MacOS users', () => {});
+      it('should show "Ctrl-shortcutKey" by default');
```

Comment:
It looks like these tests are unfinished.

Reply:
Derp. Fixed.